src/ngx_http_lua_shdict.c

```diff
@@ -2983,6 +2983,34 @@ ngx_http_lua_ffi_shdict_set_expire(ngx_shm_zone_t *zone, u_char *key,
 
     return NGX_OK;
 }
+
+
+void
+ngx_http_lua_ffi_shdict_capacity(ngx_shm_zone_t *zone,
+    size_t *capacity_bytes)
+{
+    *capacity_bytes = zone->shm.size;
+}
+
+
+#    if nginx_version >= 1011007
+void
```

Comment:
Can we return the size directly without using pointers? This would make the FFI side code simpler.

Reply:
done